sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/ConditionalSimplificationCodeFixProvider.cs

```diff
@@ -117,10 +117,9 @@ public class ConditionalSimplificationCodeFixProvider : SonarCodeFixProvider
             ExpressionSyntax condition, ExpressionSyntax compared, SemanticModel semanticModel, SyntaxAnnotation annotation,
             bool isNullCoalescing)
         {
-            var return1 = statement1 as ReturnStatementSyntax;
             var return2 = statement2 as ReturnStatementSyntax;
 
-            if (return1 != null && return2 != null)
+            if (statement1 is ReturnStatementSyntax return1 && return2 != null)

```

Comment:
Same comment as above - looks odd changing _return1_ but not _return2_.

Reply:
Automatic code fix on all instances.